### ISSUE TITLE
Let the user choose between 11kw and 22kw model on setup

### ISSUE THREAD

**Issue by syssi** [syssi/homeassistant-goecharger-mqtt#66]

**Comment by PoudNL** [comment]
Still interested in this feature!
I would love to limit the requested current value. Any update if this is possible or some workaround for now?

**Comment by marin849** [comment]
I agree. I think the only change needed is to set "Requested current" allowed values to 0-16 or 0-32A. go_echarger_xxxxx_amp
